File: src/java/me/indexyz/strap/object/Message.java
package me.indexyz.strap.object;

import java.util.List;

public class Message {
    public long message_id;

    public User from;
    public Chat chat;

    public long date;
    public String text;

    public User forward_from;
    public Chat forward_from_chat;
    public int forward_from_message_id;
    public String forward_signature;
    public int forward_date;
    public Message reply_to_message;
    public int edit_date;
    public String media_group_id;
    public String author_signature;

    public List<User> new_chat_members;
}

File: src/kotlin/me/indexyz/strap/objects/Message.kt
package me.indexyz.strap.objects

data class Message(
        val message_id: Long = 0,

        val from: User? = null,
        val chat: Chat? = null,

        val date: Long = 0,
        val text: String? = null,

        val forward_from: User? = null,
        val forward_from_chat: Chat? = null,
        val forward_from_message_id: Int = 0,
        val forward_signature: String? = null,
        val forward_date: Int = 0,
        val reply_to_message: Message? = null,
        val edit_date: Int = 0,
        val media_group_id: String? = null,
        val author_signature: String? = null,

        val new_chat_members: List<User>? = null
)